configs/francoischalifour.json

```diff
@@ -14,16 +11,15 @@
     "lvl3": ".post h4",
     "lvl4": ".post h5",
     "lvl5": ".post h6",
-    "text": ".post p, .post li, header p"
+    "text": ".post p, .post li"
   },
   "only_content_level": true,
   "scrap_start_urls": false,
   "conversation_id": [
     "547690260"
   ],
   "selectors_exclude": [
-    "article aside",
-    "article p span"
+    ".post p span"
```

Comment:
I added more specificity with the `.post` class.

Reply:
Great